monkey/monkey_island/cc/resources/auth/registration.py

```diff
@@ -31,3 +47,7 @@ def _get_user_credentials_from_request(request):
     password_hash = password_utils.hash_password(password)
 
     return UserCreds(username, password_hash)
+
+
+def _drop_mongo_db():
+    mongo.db.command("dropDatabase")
```

Comment:
why not `mongo.drop_database('db')`? Why are we even dropping the whole database if we're going to import the exact same mitigations?